react/features/conference/components/native/Conference.js

```diff
@@ -176,6 +179,18 @@ class Conference extends AbstractConference<Props, *> {
 
     _onHardwareBackPress: () => boolean;
 
+    _onDimensionChange: () => void;
+
+    /**
+     * Changes the value of the width and height.
+     *
+     * @private
+     * @returns {void}
+     */
+    _onDimensionChange() {
+        this.props.dispatch(clientResized(Dimensions.get('window').width, Dimensions.get('window').height));
```

Comment:
We have this already: https://github.com/jitsi/jitsi-meet/tree/9f9e192c3ca795826b08e0bfca79a2ea081844e5/react/features/base/responsive-ui so I think all we need to do is save the height and width in redux (as web does) and we could use that instead of computing them here.

Reply:
Sorry, missed that...